docs/hosting.md

```diff
@@ -2,4 +2,91 @@
 
 The process of hosting a Robyn app on various cloud providers.
 
-Coming Soon....
+
+### Railway
+
+To deploy an app on Railway is necessary to have a Github account, because the apps are deploying from a Github repository. 
+
+The repository we will be deploy is a simple app that display a "Hello World" message and serve a HTML file.
+
+Directory structure:
+
+```
+app folder/
+  main.py
+  requirements.txt
+  index.html
+
+```
+
+Prerequisites:
+
+- Github Account
+
+This is a template of the Robyn app we will be deploy.
+
+main.py
+```
+from robyn import Robyn, static_file
+
+
+app = Robyn(__file__)
+
+
+@app.get("/hello")
+async def h(request):
+    print(request)
+    return "Hello, world!"
+
+@app.get("/")
+async def get_page(request):
+    return static_file("./index.html")
+
+
+app.start(url="0.0.0.0", port=PORT)    
+
+```
+
+index.html
+
+```
+<h1> Hello World, this is Robyn framework! <h1>
+
+```
+
```

Comment:
### Exposing Ports

Let's add a header like the one above to segregate sections better?